test/lp/ensure_integer_bounds.jl

```diff
@@ -0,0 +1,41 @@
+function ensure_integer_bounds_tests(T::Type)
+
+    # Build the following model
+    #=
+        min     x + y + z + w
+        s.t.   -10 ⩽ 2 x + 3 y + 4 z + w ⩽ 10
+                -1.5 ⩽ x ⩽ 1.6
+                -2.5 ⩽ y ⩽ 2.1
+                -1.3 ⩽ z ⩽ 3
+                0 ⩽ w ⩽ 0
+                x is integer, z, w are binary  =#
+    C = T.([1, 1, 1, 1])
+    lc = T.([-1.5, -2.5, -1.3, 0])
+    uc = T.([1.6, 2.1, 3, 0])
+    lr = T.([-10])
+    ur = T.([10])
+    varTypes = [MathOptPresolve.GENERAL_INTEGER, MathOptPresolve.CONTINUOUS,
+                MathOptPresolve.BINARY]
```

Comment:
How come `varTypes` only has `3` elements and we're not getting an error when loading the problem?

Reply:
This is indeed odd, would have thought this would catch it: https://github.com/mtanneau/MathOptPresolve.jl/blob/5a64d16d2e5cfb2a0cdde420595a0679282f0a72/src/problem_data.jl#L122-L124

Reply:
Did we figure out why this is happening?

Reply:
IDK

Reply:
In the first version, the test function wasn't called in runtests.jl.